Solutions/CohesitySecurity/build.ps1

```diff
@@ -0,0 +1,13 @@
+$SolutionJsonPath = "Solution_CohesitySecurity.json"
```

Comment:
Add one more step here - creating packages for the Azure functions. Also, make the script commit this package to github, so the end user can deploy it to Azure right away using func command and don't care of building them (see https://github.com/cohesity/Azure-Sentinel/tree/master/Solutions/CiscoUmbrella/Data%20Connectors for example)